README.md

```diff
@@ -384,6 +384,7 @@ Name | Website | Region
 [Kentik](/company-profiles/kentik.md) | https://www.kentik.com/careers | USA, Europe, APAC
 [Khan Academy](/company-profiles/khan-academy.md) | https://www.khanacademy.org/ | USA
 [KickBack Rewards Systems](/company-profiles/kickback-rewards-systems.md) | https://careers.kickbacksystems.com | USA
+[Kyndryl](/company-profiles/kyndryl.md) | https://www.kyndryl.com/ | Worldwide
```

Comment:
This should be above `[Kuali](/company-profiles/kuali.md) | https://kuali.co/ | USA`

Reply:
I fixed it. Should be below shouldn't it?

Reply:
@zoltanvin Woops, yes!